frontend/src/metabase/query_builder/actions/timelines.js

```diff
@@ -14,6 +14,14 @@ export const selectTimelineEvents = createAction(SELECT_TIMELINE_EVENTS);
 export const DESELECT_TIMELINE_EVENTS = "metabase/qb/DESELECT_TIMELINE_EVENTS";
 export const deselectTimelineEvents = createAction(DESELECT_TIMELINE_EVENTS);
 
+export const HIDE_TIMELINE_EVENTS = "metabase/qb/HIDE_TIMELINE_EVENTS";
+export const hideTimelineEvents = createAction(HIDE_TIMELINE_EVENTS);
+
+export const SHOW_TIMELINE_EVENTS = "metabase/qb/SHOW_TIMELINE_EVENTS";
+export const showTimelineEvents = createAction(SHOW_TIMELINE_EVENTS);
+
+export const CREATE_TIMELINE_EVENT = "metabase/entities/timelineEvents/CREATE";
```

Comment:
This is a default action type coming from entities.

I think we can use a ready-made constant for this, and I looked for an example, but so far I haven't found it.

Is there a ready-made?

Reply:
`TimelineEvents.actionTypes.CREATE` or something like that

Reply:
Yup, that. Thank you